sneck/Deck.py

```diff
@@ -295,47 +386,78 @@ class Type(Enum):
-                              api.request(f'boards/{bid}/stacks/{self.__stack_id}/cards/{self.__id}/attachments')}
-                              if card['attachmentCount'] > 0 else {})
-        
-        print(f'[UPDATE] Created card {self.__id}.')
+        if card is not None and bid is not None and labels is not None and users is not None:
+            # IDs of the parents of the card
+            self.__board_id = bid
+            self.__stack_id = card['stackId']
+
+            # References to the label and user dictionaries and the API.
+            self.__users_dict = users
+            self.__labels_dict = labels
+            self.__api = api
+
+            # Attributes of the card
+            self.__id = card['id']
+            self.__tag = card['ETag']
+            self.__type = self.__types[card['type']]
+            self.__order = card['order']
+            self.__title = card['title']
+            self.__description = card['description'].strip()
+            self.__labels = (None if card['labels'] is None else 
+                            [labels[label['ETag']] for label in card['labels']])
+            self.__archived = card['archived']
+            self.__unread_comments_count = card['commentsUnread']
+            self.__overdue = card['overdue']
+
+            # Timestamps
+            self.__creation_time = dt.fromtimestamp(card['createdAt']).astimezone(tz.utc)
+            self.__last_edited_time = dt.fromtimestamp(card['lastModified']).astimezone(tz.utc)
+            self.__deletion_time = (dt.fromtimestamp(card['deletedAt']).astimezone(tz.utc)
+                                    if card['deletedAt'] != 0 else None)
+            self.__card_due_time = (None if card['duedate'] is None else
+                                    dt.strptime(card['duedate'], '%Y-%m-%dT%H:%M:%S%z').astimezone(tz.utc))
+
+            # Users
+            self.__assigned_users = (None if card['assignedUsers'] is None else 
+                                    [users[assignee['participant']['uid']] for assignee in card['assignedUsers']])
+            self.__owner = (users[card['owner']['uid']] if isinstance(card['owner'], dict) else
+                            users[card['owner']])
+            self.__last_editor = users[card['lastEditor']] if card['lastEditor'] is not None else None
+
+            # Card attachments
+            self.__attachment_count = card['attachmentCount']
+            self.__attachments = ({attachment['id']: DeckAttachment(attachment, self.__stack_id, self.__board_id, users) for attachment in
+                                api.request(f'boards/{bid}/stacks/{self.__stack_id}/cards/{self.__id}/attachments')}
+                                if self.__attachment_count is not None and self.__attachment_count > 0 else {})
+                                
+            print(f'[UPDATE] Created card {self.__id}.')
+        else:
```

Comment:
It might be beneficial to add a `new_card` boolean flag and set it to true when turning into this branch (and false otherwise), The reason being that functions like `set_title` should ideally automatically push the changes as soon as the client calls them, but of course only if the card already exists in the web application, instead when this branch is selected nothing should happen as the object is temporary and just used to provide data for insertion.